src/main/java/org/testeditor/fixture/host/util/MultilineReader.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.host.util;
+
+import org.testeditor.fixture.host.Locator;
+
+import java.util.List;
+
+/**
+ * This is a utility class for reading multiple lines on a Mainframe Screen. For
+ * reading multiple lines, we need a start point and an end point, these start
+ * and end points can received through a start row and start column and end row
+ * and end column.
+ *
+ */
+public class MultilineReader {
+
+    /**
+     * 
+     * @param lines
+     *            the rows which should be read.
+     * @param locator
+     *            see {@link Locator} the start and endpoint representation of
+     *            String to be read.
+     * @return the excerpted String from start to endpoint
+     */
+    public static String read(List<String> lines, Locator locator) {
```

Comment:
I'm not a fan of using static methods. You could merge all those readers into a single class `LineReader` and instantiate it once and then call the individual methods. You could also have a generic `read(...)` that checks the locator and then determines which read operation should be performed.